yldio/asap-hub
# testing branch teardown

- CRN-1052 News Data Provider
- fix lint
- fix fixture
- fix lint again
- unskip tests
- fix dependency @contentful/rich-text-types
- fix test
- ignore next
- remove unnecessary part
- fix ts error
- readding ignore next
- graphql approach
- wip fetch query
- working query without filters
- add frequency filter
- title filter
- fetch by id
- fix tests
- tidy up
- remove contentful
- fix lint
- fix versions
- prettier
- tidy up 2
- test news data provider app
- error test
- review: extract shared inferfaces (NewsDataProvider and FetchNewsProviderOptions)
- news data providers classes implement NewsDataProvider
- review: fix test - data provider selected using env var
- set up env vars
- refactor filter using TS
- fix package step
- fix package step 2
- updated schema
- update type for optional input properties
- review: add libs.newsDataProvider || to newsDataProvider
- review: isContentfulEnabled equals true check in config
- fix news data provider test
- review: app test not using jest.mock
- review: remove conditional in fixture to test thumbnail url
- trying to fix build/crn-sls-package step
- trying to fix build/crn-sls-package
- fix package action
- fix workflow secrets
- add test migration WIP
- modify teardown WIP
